tests/local/test_ScraperExtractFromTextTest.py

```diff
@@ -0,0 +1,68 @@
+import json
+import os
+import unittest
+from glob import glob
+from typing import List
+
+from juriscraper.lib.importer import build_module_list
+
+
+class ScraperExtractFromText(unittest.TestCase):
+    """Adds specific tests to specific courts that are more-easily tested
+    without a full integration test.
+    """
+
+    def run_tests_on_module_str(self, module_str: str) -> List[str]:
+        """Finds all the $module_example* files and tests them with the sample
+        scraper.
+        """
+        return build_module_list(module_str)
```

Comment:
If this is just a wrapper for a one-liner, why bother with it at all?

Reply:
Mostly modeling from other tests

Reply:
Uh oh, that probably means past-Mike wrote those tests, huh? He's the worst.

Reply:
fair enough... ill drop this